exercises/components-in-depth/README.md

```diff
@@ -1,93 +1,92 @@
 # Components in Depth Exercise
 
-Create a component with the following behaviour
-
-![](./preview.gif)
-
 ## Step 1
 
-Using the `angular-cli` create a new project called `components-in-depth` inside of the `training` folder.
+Using the `angular-cli` create a new project called `components-in-depth` inside 
+of the `training` folder.
 
 ## Step 2
 
-Create a child component called `<app-timer>` with three buttons.
-
-- **Start timer:** Starts the timer
-- **Stop timer:** Stops the timer
-- **Reset timer:** Resets the timer
+Use the `angular-cli` to create the components by running the following 
+commands:
 
-Using `<ng-content>` with `select`, allow each button to receive its own custom label
+```
+ng generate component button-grouping
+ng generate component counter-display
+```
```

Comment:
I like to avoid giving them a full solution. We want them to remember the command or think about it.  Maybe we can add a hint instead saying something like: "use the `generate` option with ng to create the two components?